functions_intro/task3.js

```diff
@@ -0,0 +1,16 @@
+function checkOrder(available, ordered) {
```

Comment:
now you return console.log()

Reply:
and console.log returns undefined

Reply:
so your function `checkOrder` in fact returns undefined

Reply:
nut it should return string. Just remove console.log call from your function and return strings you have

Reply:
Got it! String is added to return and the result of the function is shown via console.log